gems/sorbet-runtime/lib/types/enum.rb

```diff
@@ -280,14 +280,18 @@ def _bind_name(const_name)
 
   sig {returns(T::Boolean)}
   def self.started_initializing?
-    @started_initializing = T.let(@started_initializing, T.nilable(T::Boolean))
-    @started_initializing ||= false
```

Comment:
I am not sure if this was an attempt to memoize the value or a short way to give it an initial value, but the ivar would always be reassigned to if it was not set to `true`.

Reply:
This was due to all approach of typing that needed this